prow/istio-pilot-e2e-auth-v1alpha1.sh

```diff
@@ -56,7 +56,7 @@ HUB="gcr.io/istio-testing"
 cd ${GOPATH}/src/istio.io/istio
 
 # Run tests with auth disabled
-make depend e2e_pilot HUB="${HUB}" TAG="${GIT_SHA}" TESTOPTS=" -mixer=true -use-sidecar-injector=true -use-admission-webhook=false -auth_enable=false -v1alpha3=true -v1alpha1=false"
+#make depend e2e_pilot HUB="${HUB}" TAG="${GIT_SHA}" TESTOPTS="-mixer=true -use-sidecar-injector=true -use-admission-webhook=false -auth_enable=false -v1alpha3=false"
```

Comment:
comment on why the first test is "disabled". if it's split and copied to other files, please just remove the test.